src/app.js

```diff
@@ -1,3 +1,38 @@
+/* eslint-disable no-console */
+/* eslint-disable max-len */
 'use strict';
 
-// Write your code here
+const generateRandomNumber = require('./modules/generateRandomNumber');
+const getBullsAndCows = require('./modules/generateRandomNumber');
+
+const readline = require('readline');
+const checkIsValidUserInput = require('./modules/checkIsValidUserInput');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const randomNumber = generateRandomNumber();
+
+const game = () => {
+  terminal.question('guess the number from 1234 to 9876 composed of four different numbers', (number) => {
+    if (checkIsValidUserInput(Number(number))) {
```

Comment:
i think smoother will be to check at start 
```
if (!checkIsValidUserInput(Number(number))) {
     console.log('the number you entered is incorrect. ...');
      game();
}
```

and then just do rest of logic. In this case u ommit that one block with if and else